package.json

```diff
@@ -366,6 +366,11 @@
           "type": "boolean",
           "default": true,
           "description": "Specifes whether OmniSharp should use VS Code editor settings for C# code formatting (use of tabs, indentation size)."
+        },
+        "omnisharp.format.enable": {
```

Comment:
Do note that, while this project lives in the OmniSharp organization on GitHub, it is the official C# for VS Code that is owned and built by the C# team at Microsoft.